app/src/main/java/com/dessalines/thumbkey/utils/AbbreviationManager.kt

```diff
@@ -0,0 +1,59 @@
+package com.dessalines.thumbkey.utils
+
+import android.content.Context
+import android.util.Log
+import com.dessalines.thumbkey.db.AbbreviationDao
+import com.dessalines.thumbkey.db.AppDB
+
+private const val ABBR_TAG = "AbbreviationManager"
+
+class AbbreviationManager(
```

Comment:
I don't understand the point of this at all. You should just be passing the `AbbreviationViewModel` to any screen that needs it, and reading the abbreviations at the top of that file.